pkg/antctl/output/output.go

```diff
@@ -148,6 +148,16 @@ func YamlOutput(obj interface{}, writer io.Writer) error {
 	return nil
 }
 
+// RawOutput is an output formatter whose output is similar to fmt.Print(responseString)
+// to better display multiple-line string responses.
+func RawOutput(obj interface{}, writer io.Writer) error {
```

Comment:
Hi @Atish-iaf  Could you help to check latest antctl binary to see if the issue reported in #3426 is still valid? I think that @jianjuns refined the output format for json and yaml, it should have no issue for json output.

Reply:
ignore me, I just found following discussion in the issue is not about json or yaml output anymore..